CHANGELOG.md

```diff
@@ -55,6 +55,13 @@
 
 #### Under the hood
 
+- New `v1` versions of `CustomResourceDefinitions` introduced for KIC 2.0 are now
+  backwards compatible with the previous `v1beta1` CRD definitions. In practice
+  the upgrade process should be seamless for end-users (e.g. `kubectl apply -f`).
```

Comment:
kubectl apply -f what ? what resources are contained within the yaml ?

Reply:
You're right, let's clarify:
```suggestion
  the upgrade process should be seamless for end-users (e.g. `kubectl apply -f <NEW CRDS>`).
```